lib/services/elasticsearch.js

```diff
@@ -1004,6 +1004,7 @@ class ElasticSearch extends Service {
     try {
       if (!_.isEmpty(mappings)) {
         await this.updateMapping(index, collection, mappings);
+        await this.updateSearchIndex(index, collection);
```

Comment:
This might be costly. When we discussed this item with @Aschen , we wondered if it wouldn't be better to only perform this operation if there is at least one `dynamic: 'false'` property set in the mappings (prior to update)